eel/edge.py

```diff
@@ -1,20 +1,48 @@
-import platform
+import os
 import subprocess as sps
 import sys
 from typing import List
 
 from eel.types import OptionsDictT
 
-name: str = 'Edge'
+name: str = "Edge"
 
 
-def run(_path: str, options: OptionsDictT, start_urls: List[str]) -> None:
-    cmd = 'start microsoft-edge:{}'.format(start_urls[0])
-    sps.Popen(cmd, stdout=sys.stdout, stderr=sys.stderr, stdin=sps.PIPE, shell=True)
+def run(path: str, options: OptionsDictT, start_urls: List[str]) -> None:
+    if path != "edge_legacy":
```

Comment:
I would propose passing back the string `'start microsoft-edge'` from _find_edge_win(). For two reasons:

1. This ensures that what is returned by _find_edge_win() is always executable in principle if it is not `None`.
2. Users may locally introduce an alias for a binary, e.g. when they have several browser versions running, that they might reasonably call *edge_legacy*, so that any errors (even if unlikely) caused by a name conflict on the system would be untraceable.